### PR TITLE
Make amp-accordion accessible

### PR DESCRIPTION
**Pull request by sriramkrish85**

### CODE REVIEW

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ class AmpAccordion extends AMP.BaseElement {
       log.warn(TAG, `Experiment ${EXPERIMENT} disabled`);
       return;
     }
-    this.sections_.forEach(section => {
+    this.element.setAttribute('role', 'tablist');
```
Comment (dvoytenko) [inline extensions/amp-accordion/0.1/amp-accordion.js]: Please ensure that you also add `aria-expanded` to the `tabpannel` (along with the existed `expanded`) as described here: https://www.w3.org/TR/wai-aria/roles#tablist

Comment (dvoytenko) [inline extensions/amp-accordion/0.1/amp-accordion.js]: You will also have to set the initial value on whichever panels have `enabled` set by the publisher.

Comment (sriramkrish85) [inline extensions/amp-accordion/0.1/amp-accordion.js]: Done!

**Comment by sriramkrish85** [comment]
PTAL

File: extensions/amp-accordion/0.1/amp-accordion.js [hunk 2]
```diff
@@ -58,14 +59,26 @@ class AmpAccordion extends AMP.BaseElement {
...
       const content = sectionComponents_[1];
       header.classList.add('-amp-accordion-header');
+      header.setAttribute('role', 'tab');
       content.classList.add('-amp-accordion-content');
+      content.setAttribute('role', 'tabpanel');
+      content.setAttribute(
+          'aria-expanded', section.hasAttribute('expanded').toString());
+      let contentId = content.getAttribute('id');
+      if (!contentId) {
+        contentId = this.element.id + '_content_' + index;
```
Comment (dvoytenko) [inline extensions/amp-accordion/0.1/amp-accordion.js]: Add `AMP_` prefix to this ID to make it clear that we added it.

Comment (dvoytenko) [inline extensions/amp-accordion/0.1/amp-accordion.js]: That means that sometimes it might be `AMP_AMP_***` - but that's ok.

**Comment by dvoytenko** [comment]
LGTM

File: extensions/amp-accordion/0.1/amp-accordion.js (hunk 2 above)
Comment (sriramkrish85) [inline extensions/amp-accordion/0.1/amp-accordion.js]: Done

**Pull request merged**